### DISCUSSION TITLE
Copilot: The model returned successful but did not contain any response text.

### DISCUSSION THREAD

**Discussion by fewllet** [community/community#77245] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello,
I get the answer to every question: **The model returned successful but did not contain any response text.**

Restarting VSCode did not help.
Anyone else having this problem?

Thanks

**Comment by tonylucas1870** [comment]
I am as well, have updated to latest version and still the same, presume it's an issue on the server side.

**Comment by afortuna97** [comment]
Same here, and i saw that chat gpt too is off which could be related

**Comment by fewllet** [comment]
Well something similar happened this time last week. The problem was on the server and it took a few hours to get it up and running, but the replies were crashing on another error.

**Comment by fewllet** [comment]
Here we go :) it's back online
